circle.yml

```diff
@@ -15,15 +16,16 @@ dependencies:
     - "~/.cabal"
 
   override:
-    - mkdir -p /home/ubuntu/.go_workspace/src/github.com/Originate
-    - cp -r /home/ubuntu/git-town /home/ubuntu/.go_workspace/src/github.com/Originate/gt
-    - cd /home/ubuntu/.go_workspace/src/github.com/Originate/gt && go install
-    - rm -rf /home/ubuntu/.go_workspace/src/github.com/Originate/gt
+    - mkdir -p "$(dirname "$GO_PROJECT")"
+    - if [[ ! -e "$GO_PROJECT" ]]; then ln -s $HOME/git-town "$GO_PROJECT"; fi
+    - cd $GO_PROJECT && go install && go get gopkg.in/alecthomas/gometalinter.v1
```

Comment:
Okay, we can give it a try. Fumbling around with Circle is extremely slow and frustrating, so I want to make small baby steps on very focussed PRs in this area. This PR is for getting GoMetaLinter working, and it is currently working. Let's improve the Circle directory setup in another PR tomorrow. I have created #796 to track this issue.